### PR TITLE
feat: container for stories, force icons to use FontAwesome and add a…

### PR DESCRIPTION
**Pull request by vhu-axelor**
…rgs when needed
RM#72169

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import {Platform} from 'react-native';
+import {Platform, View} from 'react-native';
```
Comment (lme-axelor) [inline .storybook/preview.js:2]: View is not used here

File: packages/ui/stories/molecules/LabelText.stories.tsx [hunk 2]
```diff
@@ -17,35 +17,57 @@
...
+    },
   },
-});
+);
+
+// value,
+// size = 12,
+// color,
+// iconName = null,
+// onlyOneLine = false,
```
Comment (lme-axelor) [inline packages/ui/stories/molecules/LabelText.stories.tsx:73]: 🤔

File: packages/ui/stories/molecules/RadioSelect.stories.tsx [hunk 3]
```diff
@@ -29,6 +29,8 @@ storiesOf('ui/molecules/RadioSelect', module).add(
           {id: '0', title: 'Option 1'},
           {id: '1', title: 'Option 2'},
         ]}
+        direction="row"
+        onChange={() => console.log('Item has changed.')}
```
Comment (lme-axelor) [inline packages/ui/stories/molecules/RadioSelect.stories.tsx:33]: Why ? direction default value is defined in args and onChange is not useful in storybook

File: packages/ui/stories/molecules/StarScore.stories.tsx [hunk 4]
```diff
@@ -28,9 +27,7 @@ stories.add(
   'Default',
   args => {
     return (
-      <View style={styles.container}>
-        <StarScore {...args} color={lightTheme.colors[args.color]} />
-      </View>
+      <StarScore color={lightTheme.colors[args.color]} score={3} {...args} />
```
Comment (lme-axelor) [inline packages/ui/stories/molecules/StarScore.stories.tsx:30]: score is already defined in default value

File: packages/ui/stories/organisms/DropdownMenuItem.stories.tsx [hunk 5]
```diff
@@ -17,42 +17,58 @@
...
+        type: 'string',
+        defaultValue: 'truck',
+        control: {type: 'text'},
+      },
+      color: {
+        control: {
+          type: 'color',
+        },
+        defaultValue: '#000000',
+      },
```
Comment (lme-axelor) [inline packages/ui/stories/organisms/DropdownMenuItem.stories.tsx:47]: color should be defined with theme

File: packages/ui/stories/organisms/FloatingButton.stories.tsx [hunk 6]
```diff
@@ -48,16 +48,33 @@ const actions = [
 
 storiesOf('ui/organisms/FloatingButton', module)
   .addDecorator(story => <View style={styles.decorator}>{story()}</View>)
```
Comment (lme-axelor) [inline packages/ui/stories/organisms/FloatingButton.stories.tsx:50]: I thought you said decorator was not needed anymore ? 🤔

Comment (vhu-axelor) [reply packages/ui/stories/organisms/FloatingButton.stories.tsx:50]: If I don't leave this decorator this component would appear on the right, that's why

**Pull request merged**